### PR TITLE
remove rake tasks from the adapter

### PR DESCRIPTION
**Pull request by seuros**
PS: We don't need Appraisal as we are not testing multiple active-record version anymore .

### CODE REVIEW

**Comment by keithdoggett** [comment]
Thanks for working on this! Can you actually add the appraisal stuff back? Currently, we're only testing this against ActiveRecord 7.0, but when ActiveRecord 7.1 is released this version may be compatible and I'd like to have that infrastructure set up already.

**Comment by seuros** [comment]
@keithdoggett From my experience working on ActiveRecord adapters, i think it better if we keep 8.x.x for rails 7.0 and we release 9.x for 7.1 because there will be changes in syntax . It a bad pattern to check the version on runtime for an adapter that is used 90% of the time.

We already have this logic in previous versions starting from activerecord 4.2 .

In 5.x we supported 2 versions, but you can see deprecation warning in the activerecord 5.2.

If you think that the setup still needed, i will push back the files.

File: Gemfile [hunk 1]
```diff
@@ -5,5 +5,6 @@ gemspec
 
 gem "pg", "~> 1.0", platform: :ruby
 gem "activerecord-jdbcpostgresql-adapter", platform: :jruby
+gem "activerecord", "~> 7.0.0.alpha2"
```
Comment (keithdoggett) [inline Gemfile:8]: we should bump this to rc1 now.

File: test/attributes_test.rb [hunk 2]
```diff
@@ -71,7 +71,7 @@ def test_spatial_attributes
 
     assert_equal 3857, data.polygon.srid
     assert_equal 3857, data.path.srid
-    assert_equal data.path, data.polygon.exterior_ring
+    assert_equal data.path.to_s, data.polygon.exterior_ring.to_s
```
Comment (keithdoggett) [inline test/attributes_test.rb:74]: was there an issue with this?

File: test/basic_test.rb [hunk 3]
```diff
@@ -105,7 +105,7 @@ def test_set_point_wkt_wrong_type
   def test_default_value
     create_model
     obj = SpatialModel.create
-    assert_equal factory.point(0, 0), obj.default_latlon
+    assert_equal factory.point(0, 0).to_s, obj.default_latlon.to_s
```
Comment (keithdoggett) [inline test/basic_test.rb:108]: same here. Some sort of object comparison issues?

**Review by keithdoggett — CHANGES_REQUESTED** [review]
@seuros that's a fair point about the Appraisals and I agree we can just use more versions to properly handle each release. Especially since the releases now all seem to include fairly major changes.

Thank you for all the work here it looks very good for the most part I just have a few questions about some of the tests and want us to bump the AR version.

File: Gemfile (hunk 1 above)
Comment (seuros) [reply Gemfile:8]: done

File: test/attributes_test.rb (hunk 2 above)
Comment (seuros) [reply test/attributes_test.rb:74]: yes,  the new release of rgeo is not using the same object. but it values are the same.

Comment (keithdoggett) [reply test/attributes_test.rb:74]: hmm that seems weird. Do you know exactly what was causing the issue?

Comment (seuros) [reply test/attributes_test.rb:74]: latest version of rgeo causing the issue. I will check that upstream then i will update this test later.

**Review by keithdoggett — APPROVED** [review]

**Comment by keithdoggett** [comment]
Thanks @seuros

**Pull request merged**